treasuredocs/mkdocs.yml

```
@@ -1,4 +1,4 @@
-site_name: TreasureChest Documentation
+site_name: NexuShell Documentation
 theme: readthedocs
 repo_url: https://github.com/steviecoaster/TreausureChest
```

```suggestion
repo_url: https://github.com/steviecoaster/NexuShell
```
(Assuming you'll be updating the repo name - having said that, I guess this `repo_url` doesn't need to be correct)

---

Foldername this is in may also want changing.